mediorum/server/server.go

```diff
@@ -500,3 +505,8 @@ func (ss *MediorumServer) pollForSeedingCompletion() {
 		}
 	}
 }
+
+// discovery listens are enabled if endpoints are provided
+func (mc *MediorumConfig) discoveryListensEnabled() bool {
```

Comment:
yep! local stack / audius-docker-compose will by default use discovery listens because we set one endpoint here. This means that listens with playwright would also use this
https://github.com/AudiusProject/audius-protocol/pull/8584/files#diff-fd4d99326f8283204f3efdb378b1573335bdb9d0ef623f5343453c46ce1bbad7R23

Reply:
We'll want to be careful/sure that we update audius-docker-compose then to point to stage DN1 and prod DN1 before this gets pushed out then so listens don't break when they accidentally send to the local dev url

Reply:
oh, it'll be an empty value in audius-docker-compose so the feature will naturally be off

Reply:
this env var doesn't get baked into the image if that's what you're asking

Reply:
gotcha yeah that's what I was asking! cool, sg then